### PR TITLE
Update beats-agent-comparison.asciidoc

### PR DESCRIPTION
**Pull request by nimarezainia**
update root permissions fr the capabilities page

### CODE REVIEW

File: docs/en/ingest-management/beats-agent-comparison.asciidoc [hunk 1]
```diff
@@ -271,9 +271,9 @@ The following table shows a comparison of capabilities supported by {beats} and
 
 |Run without root on host
 |{y}
-|{n}
-|{n}
-|{fleet}-managed {agent}s require root permission, in particular for {elastic-defend}. Standalone {agent}s also require root permissions. {beats} requires root permission only if it's configured to capture data that requires that level of permission.
+|{y}
+|{y}
+|{fleet}-managed {agent}s, Standalone {agent}s and {beats} require root permission only if it's configured to capture data that requires that level of permission.
```
Comment (kilfoyle) [inline docs/en/ingest-management/beats-agent-comparison.asciidoc:276]:
```suggestion
|{fleet}-managed {agent}s, Standalone {agent}s, and {beats} require root permission only if they're configured to capture data that requires that level of permission.
```

**Review by kilfoyle — APPROVED** [review]
LGTM! Thanks Nima. I'll merge and backport.

**Review by karenzone — APPROVED** [review]
LGTM

**Comment by kilfoyle** [comment]
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.15|[](https://github.com/elastic/ingest-docs/pull/1321)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

**Pull request merged**